ziti/cmd/edge/quickstart.go

```diff
@@ -75,7 +75,7 @@ func NewQuickStartCmd(out io.Writer, errOut io.Writer, context context.Context)
 	cmd.Flags().StringVarP(&options.Username, "username", "u", "", "Username to use when creating the Ziti Edge Controller. default: admin")
 	cmd.Flags().StringVarP(&options.Password, "password", "p", "", "Password to use for authenticating to the Ziti Edge Controller. default: admin")
 
-	cmd.Flags().BoolVar(&options.AlreadyInitialized, "already-initialized", false, "Specifies the PKI does not need to be created and the db does not need to be initialized. Recommended to be combined with --home. If --home is not specified the environment will be destroyed on shutdown! default: false")
+	cmd.Flags().BoolVar(&options.AlreadyInitialized, "already-initialized", false, "deprecated and implied by --home on subsequent runs")
```

Comment:
I'd like to see an output that tells the user the environment won't be cleaned up because they're using --home and they should remove it if they care

Reply:
added info message if home is set in line 103